### PR TITLE
add restaurant routes and form and stuff

### PR DESCRIPTION
**Pull request by LucasSantoro**
gives the index, show and edit form for restaurants to start adding them. tags next

### CODE REVIEW

File: app/views/restaurants/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    Walking distance: <%= f.number_field :walking_minutes_away %>
+  </div>
+  <div>
+    Phone: <%= f.text_field :phone_number %>
+  </div>
+  <div>
+    Address: <%= f.text_field :street_address %>
+  </div>
+  <div>
+    Menu: <%= f.text_field :menu_link %>
```
Comment (coreyja) [inline app/views/restaurants/new.html.erb:15]: Why not just f.input for all of these? Not a huge deal for this project, but this skips all the wrapper stuff. Was that on purpose?

Comment (coreyja) [reply app/views/restaurants/new.html.erb:15]: Ooohhhh! It's cause we aren't using SimpleForms! Nvmd lol I'm cool with not using simple forms 💯

**Review by coreyja — APPROVED** [review]
LG

File: app/views/restaurants/new.html.erb (hunk 1 above)
Comment (LucasSantoro) [reply app/views/restaurants/new.html.erb:15]: Haha yup, didn't want to open that can of worms

**Pull request merged**